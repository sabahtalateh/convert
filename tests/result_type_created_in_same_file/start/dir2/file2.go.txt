package dir2

import (
	"github.com/sabahtalateh/convert/tests/result_type_created_in_same_file/work/dir1"
)

//go:generate convert
func Struct2FromStruct1(s1 *dir1.Struct1) *Struct2 {}
