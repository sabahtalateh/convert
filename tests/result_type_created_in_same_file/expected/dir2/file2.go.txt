package dir2

import (
	"github.com/sabahtalateh/convert/tests/result_type_created_in_same_file/work/dir1"
)

type Struct2 struct {
	Field1 string
}

func Struct2FromStruct1(s1 *dir1.Struct1) *Struct2 {
	var out *Struct2

	out.Field1 = s1.Field1

	return out
}
